js/src/components/shipping-rate-section/estimated-shipping-rates-card/getHandlers.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Internal dependencies
+ */
+import { SHIPPING_RATE_METHOD } from '.~/constants';
+import isNonFreeFlatShippingRate from '.~/utils/isNonFreeFlatShippingRate';
+
+/**
+ * @typedef { import(".~/data/actions").ShippingRate } ShippingRate
+ * @typedef { import(".~/data/actions").CountryCode } CountryCode
+ * @typedef { import("./typedefs").ShippingRateGroup } ShippingRateGroup
+ */
+
+const defaultShippingRate = {
+	method: SHIPPING_RATE_METHOD.FLAT_RATE,
+	options: {},
+};
+
+/**
+ * Get handlers for EstimatedShippingRatesCard.
+ *
+ * The handlers are to convert shipping rate group into shipping rates that can be propagated up via `onChange`.
+ *
+ * @param {Object} props
+ * @param {Array<ShippingRate>} props.value Array of individual shipping rates to be used as the initial values of the form.
+ * @param {(newValue: Array<ShippingRate>) => void} props.onChange Callback called with new data once shipping rates are changed.
+ */
+const getHandlers = ( { value, onChange } ) => {
```

Comment:
:nail_care:
Personally, I find `getHandlers` pretty vague. All I can read from that name is that it "returns something that's used to handle something" which is not much helpful.
WDYT of something more descriptive that states what functions it returns, like `getShipingRatesOperations`

Then, it's somewhat alerting to me that `getHandlers` returns on `handler` and two `get*Handlers`. `getHandler` that returns `get*Handler` isn't it `getGetHandler` ;) - that sounds quite redundant

Also, what is the value of having one function (`getHandlers`) that returns a set of three disjointed functions? They do not need access to each other or share any logic, do they?

Maybe we can just `export { getAddHandler, getChangeHandler, getDeleteHandler }` directly? Then we will save on constructing & destructing, and avoid the naming problem mentioned above. That will also make them more atomic and easier to test and maintain independently.

Reply:
:nail_care: Plus, I'd make it at least 
```suggestion
const getHandlers = ( value, onChange ) => {
```

As we call it as `getHandlers( { value, onChange } );` anyway, so constructing and deconstructing only blurs the picture for me